### DISCUSSION TITLE
্+ৗ ক্যারেক্টার দুটি ব্যবহার করে ঔ লিখতে পারছিনা। #fixed

### DISCUSSION THREAD

**Discussion by rashidi77** [OpenBangla/OpenBangla-Keyboard#256] [General]
ফিক্সড (বিজয়) লেআউট ব্যবহার করে ' ্+ৗ' ক্যারেক্টার দুটি ব্যবহার করে 'ঔ' লিখতে পারছিনা। একই লেআউট দিয়ে উইন্ডোজের অভ্র সফটওয়ারের মাধ্যমে '্+ৗ' এভাবে ঔ লিখা যায়।

সাময়িক সমাধান : Shift + Right Alt কী চেপে 'ঔ' লিখা। এটা বিজয় ইউজারদের জন্য এক ধরণের পেইন।

লেআউটটি Avro Layout Editor দিয়ে এডিট করা।
Tested on : OpenBangla Keyobard 3.00

**Comment by mominul** [comment]
#258 ওপেন করা হয়েছে implementation ট্র্যাকিং এর জন্য, ধন্যবাদ!

**Comment by NabilSnigdho** [comment]
@rashidi77 #258 implement হওয়ার আগে সাময়িক সমাধান হিসেবে আপনার লেআউটে `ৗ` এর পরিবর্তে `ৌ` ব্যবহার করতে পারেন, ওপেন বাংলার বিল্ট-ইন National লেআউটে এরকমই আছে। `ৗ`, `ৌ` এই ক্যারেক্টার দুটি এই সমস্যাটি ছাড়া অন্য ক্ষেত্রে ওপেন বাংলায় একই কাজ করে।

**Reply by rashidi77** [reply]
আমার আপাতত কাজ চলে যাচ্ছে। তবে এটা implement হলে ভালো হয়।

**Comment by mominul** [comment]
বাগটি ফিক্স করা হয়েছে, আপনাকে নতুন করে ক্লোন আপডেট ও কম্পাইল করে ইন্সটল করতে হবে।

**Reply by rashidi77** [reply]
আমার কাছে এটা তুলনামূলক বাগ ফিক্সের চাইতে ফিচার ইমপ্রুভমেন্ট মনে হয়। ফিচারটি ইমপ্লিমেন্ট করার জন্য ধন্যবাদ।

**Reply by mominul** [reply]
ধন্যবাদ আপনাকেও :blush:
